gradle.properties

```diff
@@ -1 +1 @@
-kotlin.mpp.enableGranularSourceSetsMetadata=true
\ No newline at end of file
+#kotlin.mpp.enableGranularSourceSetsMetadata=true
```

Comment:
Not being used, I do want to keep it there though so that once kotlin fixes the cinterop issues for intermediate source sets we can just comment this out.